src/main/java/org/cbioportal/service/impl/StudyViewColumnarServiceImpl.java

```diff
@@ -0,0 +1,291 @@
+package org.cbioportal.service.impl;
+
+import org.cbioportal.model.AlterationCountByGene;
+import org.cbioportal.model.CaseListDataCount;
+import org.cbioportal.model.ClinicalAttribute;
+import org.cbioportal.model.ClinicalData;
+import org.cbioportal.model.ClinicalDataCount;
+import org.cbioportal.model.ClinicalDataCountItem;
+import org.cbioportal.model.ClinicalEventTypeCount;
+import org.cbioportal.model.CopyNumberCountByGene;
+import org.cbioportal.model.GenericAssayDataCountItem;
+import org.cbioportal.model.GenomicDataCount;
+import org.cbioportal.model.PatientTreatmentReport;
+import org.cbioportal.model.GenomicDataCountItem;
+import org.cbioportal.model.Sample;
+import org.cbioportal.model.SampleTreatmentReport;
+import org.cbioportal.model.StudyViewFilterContext;
+import org.cbioportal.persistence.StudyViewRepository;
+import org.cbioportal.service.AlterationCountService;
+import org.cbioportal.service.StudyViewColumnarService;
+import org.cbioportal.service.exception.StudyNotFoundException;
+import org.cbioportal.service.treatment.TreatmentCountReportService;
+import org.cbioportal.service.util.StudyViewColumnarServiceUtil;
+import org.cbioportal.utils.config.annotation.ConditionalOnProperty;
+import org.cbioportal.web.parameter.ClinicalDataType;
+import org.cbioportal.web.parameter.CustomSampleIdentifier;
+import org.cbioportal.web.parameter.GenericAssayDataBinFilter;
+import org.cbioportal.web.parameter.GenericAssayDataFilter;
+import org.cbioportal.web.parameter.GenomicDataBinFilter;
+import org.cbioportal.web.parameter.GenomicDataFilter;
+import org.cbioportal.web.parameter.SampleIdentifier;
+import org.cbioportal.web.parameter.StudyViewFilter;
+import org.cbioportal.web.columnar.util.CustomDataFilterUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
```

Comment:
@onursumer Probably need to update your IDE

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class This should do it. Strange that we need to actually enter a number like `999` to prevent it from happening.